Remotion/Data/DomainObjects/Persistence/StorageProvider.obsolete.cs

```diff
@@ -0,0 +1,103 @@
+// This file is part of the re-motion Core Framework (www.re-motion.org)
+// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+// 
+// The re-motion Core Framework is free software; you can redistribute it 
+// and/or modify it under the terms of the GNU Lesser General Public License 
+// as published by the Free Software Foundation; either version 2.1 of the 
+// License, or (at your option) any later version.
+// 
+// re-motion is distributed in the hope that it will be useful, 
+// but WITHOUT ANY WARRANTY; without even the implied warranty of 
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the 
+// GNU Lesser General Public License for more details.
+// 
+// You should have received a copy of the GNU Lesser General Public License
+// along with re-motion; if not, see http://www.gnu.org/licenses.
+// 
+using System;
+using System.Collections.Generic;
+using Remotion.Data.DomainObjects.DataManagement;
+using Remotion.Data.DomainObjects.Mapping;
+using Remotion.Data.DomainObjects.Mapping.SortExpressions;
+using Remotion.Data.DomainObjects.Persistence.Configuration;
+using Remotion.Data.DomainObjects.Queries;
+using Remotion.Data.DomainObjects.Tracing;
+
+namespace Remotion.Data.DomainObjects.Persistence
+{
+  [Obsolete("Use IReadOnlyStorageProvider or IStorageProvider instead.", true)]
+  public abstract class StorageProvider
+  {
+    [Obsolete("Deriving from the obsolete StorageProvider class is no longer allowed. "
+              + "Derive from RdbmsProvider instead, or fully implement the IStorageProvider interface in your class.", true)]
```

Comment:
```suggestion
              + "Derive from RdbmsProvider instead, or fully implement the IStorageProvider interface in your class. (Version 7.0.0)", true)]
```

please add to the other messages.